cluster-autoscaler/cloudprovider/digitalocean/godo/databases_test.go

```diff
@@ -36,7 +36,7 @@ var db = Database{
 		SSL:      true,
 	},
 	Users: []DatabaseUser{
-		DatabaseUser{
```

Comment:
Can you either omit the cluster autoscaler changes or put them in a separate PR?